src/aks-preview/azext_aks_preview/_help.py

```diff
@@ -222,6 +222,9 @@
         - name: --workspace-resource-id
           type: string
           short-summary: The resource ID of an existing Log Analytics Workspace to use for storing monitoring data. If not specified, uses the default Log Analytics Workspace if it exists, otherwise creates one.
+        - name: --enable-msi-auth-for-monitoring
+          type: bool
+          short-summary: Sends monitoring data to Log Analytics using the cluster's assigned identity (instead of the Log Analytics Workspace's shared key).
```

Comment:
```suggestion
          short-summary: Send monitoring data to Log Analytics using the cluster's assigned identity (instead of the Log Analytics Workspace's shared key).
```